src/mapgen_valleys.cpp

```diff
@@ -418,9 +332,8 @@ void MapgenValleys::calculateNoise()
 	}
 
 	for (s32 index = 0; index < csize.X * csize.Z; index++) {
```

Comment:
I actually wasn't aware of the function `index(3)`.  This is actually pretty bad seeing as how there are many cases where a variable is named `index`, not just here.

What do you think I should do here?